.gitignore

```diff
@@ -10,6 +10,7 @@
 /test/tmp/
 /test/version_tmp/
 /tmp/
+.DS_Store
```

Comment:
Should this be part of the commit? I also hate these invisible Mac files but I think I set my system global gitignore to exclude these, since they're less specific to this project.

Reply:
Gotcha, I can do that!